File: pset1/Modulo01 - C/mario/Mario_V1.c
#include <cs50.h>
#include <stdio.h>

int main() {
    int altura, linha, espacos, hashtags;

    printf("Digite um número entre 1 e 8: ");
    scanf("%d", &altura);

    // Verifica se a altura está dentro do intervalo válido
    if (altura < 1 || altura > 8) {
        printf("Altura inválida. O número deve estar entre 1 e 8.\n");
        return 1;
    }

    for (linha = 1; linha <= altura; linha++) {
        // Imprime os espaços em branco antes dos hashtags
        for (espacos = altura - linha; espacos > 0; espacos--) {
            printf(" ");
        }

        // Imprime os hashtags
        for (hashtags = 1; hashtags <= linha; hashtags++) {
            printf("#");
        }

        printf("\n");
    }

    return 0;
}

File: pset1/Modulo06 - Python 🐍/mario/Mario_v1.py
altura = int(input("Digite um número entre 1 e 8: "))

# Verifica se a altura está dentro do intervalo válido
if altura < 1 or altura > 8:
    print("Numero inválida. O número deve estar entre 1 e 8.")
    exit(1)

for linha in range(1, altura + 1):
    # Imprime os espaços em branco antes dos hashtags
    for espacos in range(altura - linha, 0, -1):
        print(" ", end="")

    # Imprime os hashtags
    for hashtags in range(1, linha + 1):
        print("#", end="")

    print()
